objective.py

```diff
@@ -18,14 +19,30 @@ class Objective(BaseObjective):
     # URL of the main repo for this benchmark.
     url = "https://github.com/ambroiseodt/benchmark_ts_forecasting"
 
+    # List of parameters for the objective. The benchmark will consider
+    # the cross product for each key in the dictionary.
+    # All parameters 'p' defined here are available as 'self.p'.
+    # This means the OLS objective will have a parameter `self.whiten_y`.
+    parameters = {
+        # Parameters for cv-split
+        "n_splits": [5],  # number of folds in the cross-validation split
+        # Parameters for evaluate results
+        "eval_window_size": [512],  # size of the initial window for prediction
+        "fixed_window_size": [
+            True
+        ],  # If False, will increase the window by 'horizon' steps after each sub-evaluation
+        "horizon": [32],  # number of step to predict
+    }
+
     # List of packages needed to run the benchmark.
     # They are installed with conda; to use pip, use 'pip:packagename'. To
     # install from a specific conda channel, use 'channelname:packagename'.
     # Packages that are not necessary to the whole benchmark but only to some
     # solvers or datasets should be declared in Dataset or Solver (see
     # simulated.py and python-gd.py).
     # Example syntax: requirements = ['numpy', 'pip:jax', 'pytorch:pytorch']
-    requirements = []
+    install_cmd = "conda"
+    requirements = [numpy, sklearn]
```

Comment:
bad requirements specification:

`requirements = ["scikit-learn"]`

No need to specify numpy

Reply:
Fixed